code/modules/mob/living/carbon/human/life/handle_disabilities.dm

```diff
@@ -79,10 +79,17 @@
 						"why woud i take a pin pointner??", \
 						"FUCK IT; KISSYOUR ASSES GOOD BYE DEAD MEN! I AM SELFDESTRUCKTING THE STATION!!!!", \
 						"How do I set up the. SHow do I set u p the Singu. how I the scrungularity????", \
-						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!"))
+						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!", \
+						"Latex vox are cute!", \
+						"HOG", \
+						";ITS LOOSE", \
+						"I C-CUM-M-M-MM-M-MED-D-D!!", \
+						"VOX ARE POX", \
+						"my body is on the asteroi", \
```

Comment:
No banbaiting for braindamage lines, please

Reply:
This one actually has some pretty good historical significance. The lack of a D  indicates it being a reference.

Reply:
if an admin bans an alive person for saying their body is on the asteroid, they'd be a pretty dumb admin